### DISCUSSION TITLE
Trait Error Message

### DISCUSSION THREAD

**Discussion by bunjo05** [filamentphp/filament#14079] [Help]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

Am receiving this error message when I try to visit the admin panel route
<img width="1672" alt="Screenshot 2024-08-28 at 14 12 41" src="https://github.com/user-attachments/assets/00a8eaf1-f5ec-40ad-b2e4-dc758be2cf58">

Trait "Filament\Actions\Concerns\BelongsToGroup" not found

**Comment by sinnbeck** [comment]
Are you sure the actions package is installed correctly ?

Try
```bash
composer require filament/actions:"^3.2" -W
```
or try
```
rm -rf ./vendor
composer install
